src/java/com/github/rainmanwy/robotframework/sikulilib/keywords/ScreenKeywords.java

```diff
@@ -668,6 +702,48 @@ public void mouseMove() throws Exception{
         }
     }
 
+    @RobotKeyword("Mouse move"
+            + "Move the mouse pointer to the target region"
+            + "\n\n @cooridnates: cooridnates where mouse should move"
```

Comment:
Should be coordinates not cooridnates right? There are other parts that are written like this.